tests/README.md

```diff
@@ -0,0 +1,24 @@
+![Hummingbot](https://i.ibb.co/X5zNkKw/blacklogo-with-text.png)
+
+----
+
+The aim of Hummingbot Gateway Test scripts is to test endpoints related to supported decentralized protocols in an ideal order in each script.
+
+## Requirements
+
+- [Have gateway server properly configured and running](https://docs.hummingbot.io/gateway/installation/)
+- Ensure you have the following line containing your private key in the gateway's `global_conf.yml` file:
+
+```bash
+PRIVATE_KEY: "********"
```

Comment:
Don't put private key in a global config file - put it in a separate file, and check its permissions to make sure it's 0600, so it's inaccessible by other users.